src/main/java/busTickets/BusTicket.java

```diff
@@ -0,0 +1,99 @@
+package busTickets;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.time.LocalDate;
+
+public class BusTicket {
+    public static int validBusTickets = 0;
+    public static int startDateViolation;
+    public static int priceViolation;
+    public static int ticketTypeViolation;
+    public static final String maxViolation = FindMaxViolation.findMaxViolation(startDateViolation, priceViolation, ticketTypeViolation);
```

Comment:
Why these fields are final ?

We have each BusTicket as a object in our real world, each ticket shouldn't know about the max violation for example

We need here separate classes with their responsibility. BusTicket - model class, should just store the object params. BusTicketStorage - should only store all tickets and provide methods such getAllTickets, find ticket by id, etc. BusTicketValidator - should validate ticket or list of tickets and provide result of validation. etc

Reply:
Moved entirely to new class "ValidationStatistics"